.eslintignore

```diff
@@ -1 +1,3 @@
-node_modules/
\ No newline at end of file
+node_modules/
+**/mocks/**
+**.spec.*
```

Comment:
I think, it could be better to use `overrides` section of `eslintrc` to disable specific cases for those
In general, I believe we want to lint our tests and potentially even mocks

Reply:
Tests - yes, I will reenable this. Mocks - tricky, as it would require rewriting PR checker, but maybe there won't be that many changes